runtime/Python3/src/antlr4/atn/LexerATNSimulator.py

```diff
@@ -89,7 +92,6 @@ def copyState(self, simulator:LexerATNSimulator ):
         self.startIndex = simulator.startIndex
 
     def match(self, input:InputStream , mode:int):
-        self.match_calls += 1
```

Comment:
Removing a debug counter that conflicts with the ability to use `__slots__`. Nothing else references this.